html/webappapis/dynamic-markup-insertion/opening-the-input-stream/origin-check-in-document-open-document-domain-not-matching.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<title>Origin check in document.open() - document.domain not matching</title>
+<link rel="author" title="Jochen Eisinger" href="mailto:jochen@chromium.org">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#opening-the-input-stream">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/html/resources/common.js"></script>
+<body>
+<script>
+testInIFrame("/common/blank.html", (ctx) => {
+  const doc = ctx.iframes[0].contentDocument;
+  const docOpen = doc.open.bind(doc);
+  document.domain = document.domain;
+  assert_throws("SecurityError", docOpen, "Opening a document with non-matching document.domain didn't throw");
```

Comment:
It is same origin. Same origin doesn't look at the domain part of the origin tuple.